assets/js/src/core/modules/app/base-layout/main-nav/hooks/use-main-nav.ts

```diff
@@ -37,10 +37,11 @@ const addNavItemToItemList = (items: IMainNavItem[], item: IMainNavItem): void =
 
     if (existingItem === undefined) {
       existingItem = {
-        order: isCurrentItem ? item.order : 100,
+        order: isCurrentItem ? item.order : 1000,
         id: level,
         label: item.label ?? level,
         path: levels.slice(0, index + 1).join('/'),
+        dividerBottom: item.dividerBottom ?? undefined,
```

Comment:
```suggestion
        dividerBottom: item.dividerBottom ?? false,
```

If `dividerBottom` is a boolean - we should set it to false (as default case - or)?

https://github.com/pimcore/studio-ui-bundle/pull/1856/files?show-vendored-files=false#diff-5aefae5e9e9fcc0a97d0268b4a9379cc96b7afb9151dbf183385e9d404ce97e6R21

Reply:
hm my idea was that it would me omitted as a prop if its not defined just like it was originally passed in - but I found a better way that cleans it up. :) good catch !